tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects.robot

```diff
@@ -0,0 +1,439 @@
+*** Settings ***
+Documentation      Suite to test Data Science Projects (a.k.a DSG) feature
+Library            SeleniumLibrary
+Library            OpenShiftLibrary
+Resource           ../../../Resources/OCP.resource
+Resource           ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
+Resource           ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Workbenches.resource
+Resource           ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Storages.resource
+Resource           ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource
+Suite Setup        Project Suite Setup
+Suite Teardown     Project Suite Teardown
+Test Setup         Launch Data Science Project Main Page
+Test Teardown      Close All Browsers
+
+
+*** Variables ***
+${PRJ_TITLE}=   ODS-CI DS Project
+${PRJ_RESOURCE_NAME}=   ods-ci-ds-project-test
+${PRJ_DESCRIPTION}=   ODS-CI DS Project is a test for validating DSG feature
+${NB_IMAGE}=        Minimal Python
+${WORKBENCH_TITLE}=   ODS-CI Workbench 1
+${WORKBENCH_DESCRIPTION}=   ODS-CI Workbench 1 is a test workbench using ${NB_IMAGE} image to test DS Projects feature
+${WORKBENCH_2_TITLE}=   ODS-CI Workbench 2
+${WORKBENCH_2_DESCRIPTION}=   ODS-CI Workbench 2 is a test workbench using ${NB_IMAGE} image to test DS Projects feature
+${WORKBENCH_3_TITLE}=   ODS-CI Workbench 3
+${WORKBENCH_3_DESCRIPTION}=   ODS-CI Workbench 3 is a test workbench using ${NB_IMAGE} image to test DS Projects feature
+${PV_BASENAME}=         ods-ci-pv
+${PV_DESCRIPTION}=         ods-ci-pv is a PV created to test DS Projects feature
+# PV size are in GB
+${PV_SIZE}=         2
+${DC_S3_NAME}=    ods-ci-s3
+${DC_S3_AWS_SECRET_ACCESS_KEY}=    custom dummy secret access key
+${DC_S3_AWS_ACCESS_KEY}=    custom dummy access key id
+${DC_S3_ENDPOINT}=    custom.endpoint.s3.com
+${DC_S3_REGION}=    ods-ci-region
+${DC_S3_TYPE}=    Object storage
+
+
+*** Test Cases ***
+Verify Data Science Project Page Is Accessible
+    [Tags]    Smoke    ODS-1876
+    Open Data Science Projects Home Page
+    Page Should Contain Element     ${PROJECT_CREATE_BTN_XP}
+
+Verify User Can Access Jupyter Launcher From DS Project Page
+    [Tags]    Smoke    ODS-1877
+    Open Data Science Projects Home Page
+    Page Should Contain Element     ${SPAWNER_LINK}
+    Click Element    ${SPAWNER_LINK}
```

Comment:
Where is ${SPAWNER_LINK} defined? I don't see it among the variables

Reply:
it is defined in [Projects.resource](https://github.com/red-hat-data-services/ods-ci/pull/583/files#diff-a8b8c5b4f8450479e22b16c19126ce9d4bd3856624683bc963bdbfcabbc40c24R19:~:text=%24%7BSPAWNER_LINK%7D%3D%20%20%20%20%20%20%20%20xpath%3D//a%5Btext()%3D%22Launch%20Jupyter%22%5D)

Reply:
Mh, I think I would've defined them in this file, but not a big deal. lgtm.